CONTRIBUTING.md

```diff
@@ -20,7 +20,7 @@ Our known dependencies are:
     - cmake
     - libdev-ssl
 - SQL Compiler
-  - a Java Virtual Machine (at least Java 8)
+  - a Java Virtual Machine (at least Java 19)
```

Comment:
"at least" is not great if this cuts out possible users and contributors...

Reply:
Why would this cut out possible users/contributors? This is a compiler application, not library code. Users also need a rust compiler to use/contribute to this compiler, so installing a new JDK shouldn't be an issue for them.